filechecker.py

```diff
@@ -0,0 +1,17 @@
+import os
+import argparse
+
+def parse_arguments():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('path', type=checkpath)
+    return parser.parse_args()
+
+def checkpath(path):
+    if os.path.exists(path):
+        print("Valid path!")
+        return path
+    else:
+        raise argparse.ArgumentTypeError(f"Not a valid path")
+
+
+parsed_args = parse_arguments()
```

Comment:
Always add a new line at the end of each file (I know that it works without it, but it is a good practice, read about it).